src/popup/router/index.ts

```diff
@@ -69,7 +69,9 @@ router.beforeEach(async (to, from, next) => {
     }[POPUP_TYPE];
 
     if (name !== to.name) {
-      next({ name, params: await getPopupProps() as Dictionary });
+      const params = await getPopupProps() as Dictionary;
```

Comment:
All this params related changes can be removed after router mode will be set to `history`.